lib/flight.js

```diff
@@ -6,10 +6,32 @@ const getProp = require('dot-prop').get
 const moment = require('moment-timezone')
 
 const endpoint = 'https://data-live.flightradar24.com/clickhandler/'
+
+// const headers = {
+// 	'User-Agent': 'https://github.com/derhuerst/fetch-flightradar24-flights'
+// }
+
+/// PKO header
+/// Force a rotation of user-agent string, to avoid autmated blocking @FR24 side
+/// Data source: https://www.useragents.me/  i take manually the top 10 
+///////////////////////////////////////////////////////////////////////////////////////////
+const uas= ("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246"
+,"Mozilla/5.0 (X11; CrOS x86_64 8172.45.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.64 Safari/537.36"
+,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/601.3.9 (KHTML, like Gecko) Version/9.0.2 Safari/601.3.9"
+,"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36"
+,"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:15.0) Gecko/20100101 Firefox/15.0.1"
+,"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 Edg/114.0.1823.51"
+,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.5.1 Safari/605.1.15"
+,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.3"
+,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6,2 Safari/605.1.1"
+,"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 OPR/99.0.0.") 
+
 const headers = {
-	'User-Agent': 'https://github.com/derhuerst/fetch-flightradar24-flights'
+	'User-Agent': uas[Math.floor(Math.random()*uas.length)]
```

Comment:
This will fail if `Math.random()` returns `1`. Also, it will always use the same (random) header until the Node.js process is restarted.